### PR TITLE
Fix docs build

### PR DESCRIPTION
**Pull request by oruebel**
- Add a requirments file to specify the additional packages needed to build the docs
- Various minor fixes to the docs

### CODE REVIEW

File: requirements-doc.txt
Comment (CodyCBakerPhD) [inline requirements-doc.txt:1]: This file is already on `main`, found here: https://github.com/NeurodataWithoutBorders/nwb_benchmarks/blob/main/docs/requirements-rtd.txt

the RTD website is likewise configured to find it there

**Comment by CodyCBakerPhD** [comment]
The docs are also building fine on the web: https://nwb-benchmarks.readthedocs.io/en/latest/

and on PRs (this one being the first test of that): https://nwb-benchmarks--11.org.readthedocs.build/en/11/

File: requirements-doc.txt
Comment (oruebel) [reply requirements-doc.txt:1]: Thanks, I removed the `requirements-doc.txt` file

**Comment by oruebel** [comment]
> The docs are also building fine on the web: https://nwb-benchmarks.readthedocs.io/en/latest/

The other changes should still be merged. The "Writing Benchmarks" is missing in the current build and this PR also fixes several warnings in the build due to missing files.

**Review by CodyCBakerPhD — APPROVED** [review]

**Comment by CodyCBakerPhD** [comment]
> The other changes should still be merged. The "Writing Benchmarks" is missing in the current build and this PR also fixes several warnings in the build due to missing files.

Yep, good catch

**Pull request merged**